homework-2/script/maim.js

```diff
@@ -0,0 +1,318 @@
+window.addEventListener('DOMContentLoaded', () => {
+    
+    const tableUser = document.querySelector('.table-user');
+    const modal = document.querySelector('.modal');
+    const closeBtn = document.querySelectorAll('.fa-times');
+    const checkboxFirst = document.querySelector('.tabel-checkbox-first');
+    const valueCompany = document.querySelector('.input-company');
+    const valueName = document.querySelector('.input-name');
+    const valueAddress = document.querySelector('.input-address');
+    const updateUserBtn = document.querySelector('.btn-dark');
+    const showPostForm = document.querySelector('.form-btn');
+    const modalCreateUser = document.querySelector('.modal-create-user');
+    const postBtn = document.querySelector('.post-btn');
+    const newNameData = document.querySelector('.create-name');
+    const newCompanyData = document.querySelector('.create-company');
+    const newAddressData = document.querySelector('.create-address');
+    const newCityData = document.querySelector('.create-city');
+    const newCountyData = document.querySelector('.create-country');
+    const allCheckbox = document.querySelector('.box-checkbox');
+    const paginationPage = document.querySelector('.pagination')
+    let secondaryCheckbox;
+    let deleteBtn;
+    let checkboxBtn;
+
+    const userFilterCompany = document.querySelector('.user-company');
+    const userFilterName = document.querySelector('.user-contact');
+    const filterUserBtn = document.querySelector('.filter-user');
+    const userFilterAddress = document.querySelector('.user-address');
+
+    let modalNumber;
+    let dataCompany = [];
+    let dataName = [];
+    let dataAddress = [];
+    let dataIdUser = [];
+    let navigationBtn = [];
+
+
+    let url = 'http://localhost:3000/users';
+
+    const getAllUsers = async ( url ) => {
+        const result = await fetch( url );
+
+        if( !result.ok ) {
+            throw new Error(`Could not fetch ${url}, status: ${result.status}`)
+        }
+
+        return await result.json();
+    }
+
+
+
+    const createUser = async ( obj ) => {
```

Comment:
add, change and delete users options do not work from my side, re-check please

Reply:
method POST, PATH, DELETE, everything works for me